fictadvisor-api/src/v2/api/dtos/ComplaintDTO.ts

```diff
@@ -0,0 +1,39 @@
+import { IsNotEmpty, IsOptional, IsString, IsUUID, MaxLength, MinLength } from 'class-validator';
+import { validationOptionsMsg } from '../../utils/GLOBALS';
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+
+export class ComplaintDTO {
+  @ApiPropertyOptional({
+    description: 'Student\'s full name',
+  })
+  @IsString()
+  @IsOptional()
+  @MinLength(5, validationOptionsMsg('Full name is too short (min: 5)'))
+  @MaxLength(50, validationOptionsMsg('Full name is too long (max: 50)'))
+    fullName?: string;
+
+  @ApiPropertyOptional({
+    description: 'Student\'s group id',
+  })
+  @IsUUID()
+  @IsOptional()
+    groupId?: string;
+
+  @ApiProperty({
+    description: 'Title of complaint',
+  })
+  @IsString()
+  @IsNotEmpty(validationOptionsMsg('Title can not be empty'))
+  @MinLength(5, validationOptionsMsg('Title is too short (min: 5)'))
+  @MaxLength(100, validationOptionsMsg('Title is too long (max: 100)'))
+    title: string;
+
```

Comment:
@f-ogoog is it possible to not return errors with 
`obj.*`

like here
![image](https://github.com/fictadvisor/fictadvisor/assets/86970828/1baae1f0-9d09-4c05-9c87-b55fa06f8c9a)


but without obj. thing